actions/form/class.Instance.php

```diff
@@ -153,8 +153,7 @@ protected function initElements()
                     }
                 }
 
-                // don't show empty labels
-                if ($element instanceof tao_helpers_form_elements_Label && strlen($element->getRawValue()) === 0) {
+                if ($this->isEmptyLabel($element)) { // Don't show empty labels
```

Comment:
The method name already explains the meaning, so no need for comments :) 

```suggestion
                if ($this->isEmptyLabel($element)) {
```

Reply:
Changed, thanks! :)